src/store/builder/index.ts

```diff
@@ -15,86 +18,86 @@ export enum OperationKind { // todo migrate these to flight-monitor
 }
 
 export interface IConnectionSourceRelocateOperation {
-  kind: OperationKind.CONNECTION_SOURCE_RELOCATE,
+  kind: OperationKind.CONNECTION_SOURCE_RELOCATE;
   data: null | {
-    from: IDeepBlockExitIdWithinFlow,
-    position: IPosition, // todo: rename to startingPosition
-    to: IDeepBlockExitIdWithinFlow | null,
-  }
+    from: IDeepBlockExitIdWithinFlow;
+    position: IPosition; // todo: rename to startingPosition
+    to: IDeepBlockExitIdWithinFlow | null;
+  };
 }
 
 export interface IConnectionCreateOperation {
-  kind: OperationKind.CONNECTION_CREATE,
+  kind: OperationKind.CONNECTION_CREATE;
   data: null | {
-    source: IDeepBlockExitIdWithinFlow,
-    position: IPosition, // todo: rename to startingPosition
-    target: IBlock['uuid'] | null,
-  }
+    source: IDeepBlockExitIdWithinFlow;
+    position: IPosition; // todo: rename to startingPosition
+    target: IBlock['uuid'] | null;
+  };
 }
 
 export type SupportedOperation = IConnectionSourceRelocateOperation | IConnectionCreateOperation
 
 interface IPosition {
-  x: number,
-  y: number,
+  x: number;
+  y: number;
 }
 
-
-
 export interface IBuilderState {
-  activeBlockId: IBlock['uuid'] | null,
-
+  activeBlockId: IBlock['uuid'] | null;
+  isEditable: boolean;
   operations: {
-    [OperationKind.CONNECTION_SOURCE_RELOCATE]: IConnectionSourceRelocateOperation,
-    [OperationKind.CONNECTION_CREATE]: IConnectionCreateOperation,
-    [OperationKind.BLOCK_RELOCATE]: null,
-  }
+    [OperationKind.CONNECTION_SOURCE_RELOCATE]: IConnectionSourceRelocateOperation;
+    [OperationKind.CONNECTION_CREATE]: IConnectionCreateOperation;
+    [OperationKind.BLOCK_RELOCATE]: null;
+  };
 }
 
 export const stateFactory = (): IBuilderState => ({
   activeBlockId: null,
-
+  isEditable: null,
```

Comment:
@poojahpatil90 , let's put `true` by default. Otherwise all stories for blocks in storybook will have **disabled** block config

Reply:
Sure, I will make this change.